src/ProjectOrigin.Electricity/Production/Verifiers/Allocated.cs

```diff
@@ -1,27 +1,37 @@
 using Google.Protobuf;
-using NSec.Cryptography;
 using ProjectOrigin.Electricity.Consumption;
+using ProjectOrigin.Electricity.Extensions;
 using ProjectOrigin.PedersenCommitment;
-using ProjectOrigin.Register.StepProcessor.Interfaces;
-using ProjectOrigin.Register.StepProcessor.Models;
+using ProjectOrigin.Verifier.Utils;
+using ProjectOrigin.Verifier.Utils.Interfaces;
+using ProjectOrigin.Registry.V1;
+using System.Threading.Tasks;
 
 namespace ProjectOrigin.Electricity.Production.Verifiers;
 
-internal class ProductionAllocatedVerifier : IEventVerifier<ProductionCertificate, V1.AllocatedEvent>
+public class ProductionAllocatedVerifier : IEventVerifier<ProductionCertificate, V1.AllocatedEvent>
 {
-    public Task<VerificationResult> Verify(Register.StepProcessor.Interfaces.VerificationRequest<V1.AllocatedEvent> request)
+    private IRemoteModelLoader _remoteModelLoader;
+
+    public ProductionAllocatedVerifier(IRemoteModelLoader something)
```

Comment:
Correct, it is because the .Validate requires a bool value, look in the startup.cs class in the electricty.server 

```csharp
        services.AddOptions<RegistryOptions>()
            .Configure<IConfiguration>((settings, configuration) =>
            {
                configuration.Bind(settings);
            })
            .Validate(x => x.Verify())
            .ValidateOnStart();
```